tests/Microsoft.Bot.Builder.TestBot/Controllers/BotController.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+namespace Microsoft.Bot.Builder.TestBot.Controllers
+{
+    [Route("bot")]
+    [ApiController]
+    public class BotController : ControllerBase
+    {
+        private IBotFrameworkHttpAdapter _adapter;
+        private IBot _bot;
+
+        public BotController(IBotFrameworkHttpAdapter adapter, IBot bot)
+        {
+            _adapter = adapter;
```

Comment:
Given that this is a test bot, weird stuff can happen here. The Null-check + throw will save us debugging time down the line... 
_adapter = adapter ?? throw new ...